<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenDialog</class>
 <widget class="QDialog" name="OpenDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>120</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>80</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Select image files</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="originalLabel">
     <property name="text">
      <string>Original:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="originalPath"/>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="browseOriginalPath">
     <property name="text">
      <string>Browse...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="processedLabel">
     <property name="text">
      <string>Processed:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="processedPath"/>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="browseProcessedPath">
     <property name="text">
      <string>Browse...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
